core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java

```diff
@@ -187,16 +188,20 @@ private void createZkNamespace() {
     String schemaRegistryNamespace = "/" + schemaRegistryZkNamespace;
     schemaRegistryZkUrl = zkConnForNamespaceCreation + schemaRegistryNamespace;
 
-    ZkClient zkClientForNamespaceCreation = new ZkClient(zkConnForNamespaceCreation,
-                                                         zkSessionTimeoutMs, zkSessionTimeoutMs,
-                                                         new ZkStringSerializer());
+    ZkUtils zkUtilsForNamespaceCreation = ZkUtils.apply(
+        zkConnForNamespaceCreation,
+        zkSessionTimeoutMs, zkSessionTimeoutMs,
+        JaasUtils.isZkSecurityEnabled(System.getProperty(JaasUtils.JAVA_LOGIN_CONFIG_PARAM)));
     // create the zookeeper namespace using cluster.name if it doesn't already exist
-    ZkUtils.makeSurePersistentPathExists(zkClientForNamespaceCreation, schemaRegistryNamespace);
+    zkUtilsForNamespaceCreation.makeSurePersistentPathExists(
+        schemaRegistryNamespace,
+        zkUtilsForNamespaceCreation.DefaultAcls());
     log.info("Created schema registry namespace " +
              zkConnForNamespaceCreation + schemaRegistryNamespace);
-    zkClientForNamespaceCreation.close();
-    this.zkClient = new ZkClient(schemaRegistryZkUrl, zkSessionTimeoutMs, zkSessionTimeoutMs,
-                                 new ZkStringSerializer());
+    zkUtilsForNamespaceCreation.close();
+    this.zkUtils = ZkUtils.apply(
```

Comment:
This is accessing a Scala object's `apply` from Java. Unless there's a newer shorthand I'm not aware of, I'm pretty sure we need it.